src/routes/index.ts

```diff
@@ -280,6 +281,9 @@ router.get(config.UPDATE_CONFIRMATION_URL, authentication, updateConfirmation.ge
 router.get(config.OVERSEAS_ENTITY_QUERY_URL, authentication, overseasEntityQuery.get);
 router.post(config.OVERSEAS_ENTITY_QUERY_URL, authentication, ...validator.overseasEntityQuery, checkValidations, overseasEntityQuery.post);
 
+router.get(config.UPDATE_SIGN_OUT_URL, updateSignOut.get);
+router.post(config.UPDATE_SIGN_OUT_URL, ...validator.signOut, checkValidations, updateSignOut.post);
+
```

Comment:
could you write these routes using router.route, there are other examples in the file